src/main/java/aquality/selenium/configuration/driversettings/DriverSettings.java

```diff
@@ -39,21 +50,32 @@ void logStartArguments() {
     @Override
     public String getWebDriverVersion() {
         return String.valueOf(getSettingsFile().getValueOrDefault(
-                getDriverSettingsPath(getBrowserName()) + "/webDriverVersion", "Latest"));
+                getDriverSettingsPath("webDriverVersion"), "Latest"));
     }
 
     @Override
-    public String getSystemArchitecture() {
-        return String.valueOf(getSettingsFile().getValueOrDefault(
-                getDriverSettingsPath(getBrowserName()) + "/systemArchitecture", "Auto"));
+    public Architecture getSystemArchitecture() {
+        String strValue = String.valueOf(getSettingsFile().getValueOrDefault(
+                getDriverSettingsPath("systemArchitecture"), "Auto"));
+        return Arrays.stream(Architecture.values())
+                .filter(value -> value.name().equals(strValue))
+                .findFirst()
+                .orElse(Architecture.X32);
```

Comment:
we handle default value with "Auto" or "X32"? this is strange

Reply:
Default value the same as in .NET library - Auto, but default value in WebDriverManager lib is `DEFAULT`. So, we leave Auto as a default value for both our libraries.